util/Migrator/DbScripts/2020-11-16_00_SalesTax.sql

```diff
@@ -0,0 +1,153 @@
+IF OBJECT_ID('[dbo].[TaxRate]') IS NOT NULL
+BEGIN
+    DROP TABLE [dbo].[TaxRate]
```

Comment:
This script would not be "re-runnable" in our cloud environment, it would be bad if this got deployed a 2nd time and the table dropped... Maybe create table needs to be wrapped with a not exists instead and this statement removed (sorry I missed this before).

see: `/server/util/Migrator/DbScripts/2020-02-18_00_PolicySetup.sql` for an example of when we add tables via script.

Reply:
Good point. I was thinking about re-runnable-ness in terms of a local environment, but didn't consider a live one. Thanks for the example. Implemented as used there.